src/styles/main.scss

```diff
@@ -1,3 +1,205 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
+:root {
+  --card-main: #060b35;
+  --card-secondary: #00acdc;
+  --card-border: #e5e5e5;
+}
+
+html {
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+  }
+
+  &__link {
+    height: 60px;
+    white-space: nowrap;
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: var(--card-main);
+    font-weight: 500;
+    cursor: pointer;
+
+    &:hover {
+      color: var(--card-secondary);
+    }
+
+    &:active,
+    &--active { /* keeping these for possible tests */
+      position: relative;
+      color: var(--card-secondary);
+    }
+
+    &:active::after,
+    &--active::after { /* keeping these for possible tests */
+      content: "";
+      border-radius: 5px;
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      height: 4px;
+      background-color: var(--card-secondary);
+    }
+  }
+
+  &__item + &__item {
+    margin-left: 20px;
+  }
+}
+
+.cards-container {
+  display: flex;
+  flex-wrap: wrap;
+  width: 944px;
+  margin: 0 auto;
+  padding: 50px 40px;
+}
+
+.card {
+
+  width: 200px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid var(--card-border);
+  border-radius: 5px;
+  color: #060b35;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image-wrapper {
+    margin-bottom: 35px;
+    text-align: center;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin: 0 0 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+  }
+
+  &__review-count {
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    font-weight: 400;
+
+    &-amount {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+    }
+
+    &-desc {
+      margin: 0;
+    }
+  }
+
+  &__button-buy {
```

Comment:
If you decide to change the height of button later on this padding will mess the design. You already centered the text. Adding height will solve the problem. And add box-sizing border-box also because you added borders and it will affect the height of the card.

Reply:
Correcting